promql/durations_test.go

```diff
@@ -195,6 +195,24 @@ func TestCalculateDuration(t *testing.T) {
 			expected:        -5 * time.Second,
 			allowedNegative: true,
 		},
+		{
```

Comment:
Missing tests for min, max operator.

Reply:
There are tests in `duration_expression.test` now.